src/services/api.js

```diff
@@ -11,27 +11,45 @@ const spinner = ora({
   color: 'cyan',
 })
 
-function getApiDefaults() {
+function getDefatuls() {
+  const environmments = {
```

Comment:
Nit: Small typo in environment and getDefaults, as well as in their usage below then obviously